plextraktsync/com.github.plextraktsync.watch.plist

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Label</key>
+	<string>com.github.plextraktsync.watch</string>
+	<key>ProgramArguments</key>
+	<array>
+		<string>plextraktsync</string>
+		<string>watch</string>
+	</array>
+	<key>LimitLoadToSessionType</key>
+	<string>Aqua</string>
```

Comment:
why Aqua?

from this link `Background` seems more appropriate?
- https://stackoverflow.com/questions/9576287/what-are-the-os-x-session-types-and-what-do-they-mean

Reply:
I think I used aqua based on a old example. You are right that `Background` might be better. 

Setting a `nice` level might be a good idea though.

Reply:
Send an example for nice level. perhaps using suggest code change feature:
- https://egghead.io/lessons/github-add-suggestions-in-a-github-pr-review

Reply:
@Bas-Man I can't see the suggestion on the web, but had a glimpse of it in the email. Did you delete it?